tests/typescript/custom/typeParameters/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -108,3 +108,52 @@ type ReallyReallyReallyLongName<
 
 ================================================================================
 `;
+
+exports[`variables.ts 1`] = `
+====================================options=====================================
+parsers: ["typescript"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+const foo: SomeThing<boolean> = func();
+const bar: SomeThing<boolean, boolean> = func();
+const fooo: SomeThing<{ [P in "x" | "y"]: number }> = func();
+const baar: SomeThing<K extends T ? G : S> = func();
+const fooooooooooooooo: SomeThing<boolean> = looooooooooooooooooooooooooooooongNameFunc();
+const baaaaaaaaaaaaaaaaaaaaar: SomeThing<boolean, boolean> = looooooooooooooooooooooooooooooongNameFunc();
+const baaaaaaaaaaaaaaar: SomeThing<{ [P in "x" | "y"]: number }> = looooooooooooooooooooooooooooooongNameFunc();
+const baaaaaaaaaaaaaaaar: SomeThing<K extends T ? G : S> = looooooooooooooooooooooooooooooongNameFunc();
+const isAnySuccessfulAttempt$: Observable<boolean> = this._quizService.isAnySuccessfulAttempt$().pipe(
+  tap((isAnySuccessfulAttempt: boolean) => {
+    this.isAnySuccessfulAttempt = isAnySuccessfulAttempt;
+  }),
+);
+const isAnySuccessfulAttempt$: Observable<boolean> = this._someMethodWithLongName();
+
+=====================================output=====================================
+const foo: SomeThing<boolean> = func();
+const bar: SomeThing<boolean, boolean> = func();
+const fooo: SomeThing<{ [P in "x" | "y"]: number }> = func();
+const baar: SomeThing<K extends T ? G : S> = func();
+const fooooooooooooooo: SomeThing<boolean> = looooooooooooooooooooooooooooooongNameFunc();
+const baaaaaaaaaaaaaaaaaaaaar: SomeThing<
+  boolean,
+  boolean
+> = looooooooooooooooooooooooooooooongNameFunc();
+const baaaaaaaaaaaaaaar: SomeThing<
+  { [P in "x" | "y"]: number }
+> = looooooooooooooooooooooooooooooongNameFunc();
+const baaaaaaaaaaaaaaaar: SomeThing<
+  K extends T ? G : S
+> = looooooooooooooooooooooooooooooongNameFunc();
+const isAnySuccessfulAttempt$: Observable<boolean> = this._quizService
+  .isAnySuccessfulAttempt$()
+  .pipe(
+    tap((isAnySuccessfulAttempt: boolean) => {
+      this.isAnySuccessfulAttempt = isAnySuccessfulAttempt;
+    })
+  );
```

Comment:
Looks ugly, maybe we can break after `=`?

Reply:
I agree with you. But, I think it is unrelated to this pr.
Also, it looks like same as #2482 . I tried to fix this issue when I opened #6447 , but it is difficult...

Reply:
I think it is directly related. The point of #5684, as I understood it, was that breaking after `=` should be preferred in this case. But the band-aid solution offered by this PR is still better than no solution at all.